# Added new flag to grasp vertical resizing
# contact: Anton Rogaynis ext-anton(dot)rogaynis(at)nokia.com
--- mozilla/embedding/browser/webBrowser/nsIEmbeddingSiteWindow.idl.orig	2004-04-18 18:00:40.000000000 -0400
+++ mozilla/embedding/browser/webBrowser/nsIEmbeddingSiteWindow.idl	2007-06-03 20:07:32.000000000 -0400
@@ -82,6 +82,15 @@
     const unsigned long DIM_FLAGS_SIZE_OUTER = 4;
 
     /**
+     * Flag indicates that the vertical size of the inner area is required/specified.
+     *
+     * @see setDimensions
+     * @see getDimensions
+     * @see DIM_FLAGS_VERTICAL
+     */
+    const unsigned long DIM_FLAGS_VERTICAL = 8;
+
+    /**
      * Sets the dimensions for the window; the position & size. The
      * flags to indicate what the caller wants to set and whether the size
      * refers to the inner or outer area. The inner area refers to just
